perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -901,6 +926,46 @@ def RunMultiStreamProcesses(command):
   logging.info('Finished multi-stream read test.')
 
 
+
+def MultiStreamWriteBenchmark(results, metadata, vms, command_builder,
+                              service, bucket_name, regional_bucket_name):
+
+  """A benchmark for multi-stream write latency and throughput.
+
+  Args:
+    results: the results array to append to.
+    metadata: a dictionary of metadata to add to samples.
+    vms: the VMs to run the benchmark on.
+    command_builder: an APIScriptCommandBuilder.
+    service: an ObjectStorageService.
+    bucket_name: the primary bucket to benchmark.
+    regional_bucket_name: the secondary bucket to benchmark.
+
+  Raises:
+    ValueError if an unexpected test outcome is found from the API
+    test script.
+  """
+
+  logging.info('Starting multi-stream write test on %s VMs.', len(vms))
+
+  objects_written_file = posixpath.join(vm_util.VM_TMP_DIR,
```

Comment:
You're right, don't need it. But it'd add complexity to remove it for only the api_multistream_writes scenario. I guess there's a tiny bit of overhead writing all those names to the file, but it should be negligible, right?